Chemistry/GetGateCount/Configuration.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file defines several structures used for configuring how the rest
+// of the sample runs.
+
+using System;
+
+namespace Microsoft.Quantum.Chemistry.Samples
+{
+
+    /// <summary>
+    /// Enumeration type for choice of Hamiltonian simulation algorithm
+    /// </summary>
+    public enum HamiltonianSimulationAlgorithm
+    {
+        ProductFormula,
+        Qubitization
+    };
+    /// <summary>
```

Comment:
I would leave an extra blank line for readability.

Reply:
Good point, added those.